cinema/migrations/0003_alter_ticket_options_alter_movie_image.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.1 on 2024-12-05 19:09
+
+import cinema.models
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cinema', '0002_movie_image'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='ticket',
+            options={},
+        ),
+        migrations.AlterField(
+            model_name='movie',
+            name='image',
+            field=models.ImageField(blank=True, null=True, upload_to=cinema.models.movie_image_file_path),

```

Comment:
Ensure that `cinema.models.movie_image_file_path` is defined in your models module. This function should handle the logic for generating the file path for uploaded images, as required by the task description.